Update specificatie.md (#31)
nuts-foundation/toepassing-zorginzage-ha-vvt: specificatie.md

Shortcuts en Toekomstige ontwikkelingen:
Sommige zaken kunnen we op dit moment niet invullen zoals we willen omdat dit (om verschillende redenen) nog niet realistisch. Deze zaken plaatsen we in deze usecase buiten scope en worden later eventueel toegevoegd. Het gaat om:
*We sluiten niet aan op Mitz (maar gaan consent lokaal oplossen)

> *Er is geen goed authenticatiemiddel voor de gebruiker beschikbaar (DEZI) dus werken we met een VC waarbij de organisatie obv een UZI certificaat wordt geauthenticeerd

Reviewer: bullet 2 snap ik niet. Ik lees nu: er is (nog) geen goede persoons authenticatie mogelijk, dus authenticeren we de organisatie. Volgens mij wil je altijd beide doen, maar is persoons authenticatie nog niet beschikbaar en vertrouwen we dus tijdelijk op de zelf verklaarde gegevens van de opvragende partij (NutsEmployeeCredential).

Author: Lijkt me prima om de voorgestelde tekst over te nemen

Reviewer: het is mij niet duidelijk wat je hiermee bedoelt.

*We implementeren niet alle mogelijke ZIB’s want die zijn niet beschikbaar in bronsystemen, maar hanteren een subset

> *We passen geen DPOB security toe want dit is aanvullende beveiliging die ook in andere usecase niet ingevuld wordt

Reviewer: ```suggestion
*We passen geen DPoP security toe (RFC9449)  want dit is aanvullende beveiliging die ook in andere usecase niet ingevuld wordt
```

> *We passen geen DPOB security toe want dit is aanvullende beveiliging die ook in andere usecase niet ingevuld wordt

Reviewer: Het niet doen van *DPoP "omdat andere use-cases het ook niet doen" is een slecht argument, vooral omdat het eerder niet beschikbaar was. Hier moet goed naar gekeken worden en dan een geinformeerde keuze in worden gemaakt. 
(DPoP maakt een gelekt / gestolen access token onbruikbaar. Het zorgt dus dat de bronhouder meer zekerheid heeft over de identiteit van de partij aan wie de data verstrekt wordt. Een gelekt access token kan zonder DPoP met elk willekeurig PKIo certificaat gebruikt worden om data op te vragen.)

Author: Nieuwe tekst zoal besproken in Groenlo: DPOP wordt pas in een later stadium toegevoegd, voor nu maakt dit geen onderdeel uit van de usecase

*We verrijken / corrigeren de informatie uit de UZI credentials niet (human readable namen), dit moet bij de bron opgelost worden
*Bij gebrek aan een generieke lokalisatiedienst wordt er een workaround toegepast waarbij handmatig wordt vastgelegd waar de patient in zorg is en waar dus informatie opgehaald kan worden

## Architectuur beschrijving
### Gebruik van Nuts

> Voor deze usecase wordt gebruikt gemaakt van de Nuts infrastructuur. Specifiek wordt gebruik gemaakt van de mogelijkheden die de V6 versie van Nuts biedt en daarmee dus ook van did:web. Voor de informatie specifiek over de Nuts-laag wordt verwezen naar de officiele documentatie: https://nuts-node.readthedocs.io/en/stable/   

Reviewer: ```suggestion
Voor deze usecase wordt gebruikt gemaakt van de Nuts infrastructuur. Specifiek wordt gebruik gemaakt van de mogelijkheden die de V6.1 versie van Nuts biedt en daarmee dus ook van did:web. Voor de informatie specifiek over de Nuts-laag wordt verwezen naar de officiele documentatie: https://nuts-node.readthedocs.io/en/stable/   
```

Shortcuts en Toekomstige ontwikkelingen:
Sommige zaken kunnen we op dit moment niet invullen zoals we willen omdat dit (om verschillende redenen) nog niet realistisch. Deze zaken plaatsen we in deze usecase buiten scope en worden later eventueel toegevoegd. Het gaat om:
* We sluiten niet aan op Mitz (maar gaan consent lokaal oplossen)

> * Er is geen goed authenticatiemiddel voor de gebruiker beschikbaar (DEZI) dus werken we met een VC waarbij de organisatie obv een UZI certificaat wordt geauthenticeerd

Reviewer: ```suggestion
* Er is op dit moment geen goed authenticatiemiddel voor de gebruiker beschikbaar (DEZI) dus werken we voor nu met een VC waarbij de organisatie obv een UZI certificaat wordt geauthenticeerd. De gebruiker wordt voor nu in een door de organisatie verklaard VC meegestuurd.
```

> *We sluiten niet aan op Mitz (maar gaan consent lokaal oplossen)
> *Er is geen goed authenticatiemiddel voor de gebruiker beschikbaar (DEZI) dus werken we met een VC waarbij de organisatie obv een UZI certificaat wordt geauthenticeerd
> *We implementeren niet alle mogelijke ZIB’s want die zijn niet beschikbaar in bronsystemen, maar hanteren een subset
> *We passen geen DPoP security toe (RFC9449)  want dit is aanvullende beveiliging die ook in andere usecase niet ingevuld wordt
> *We verrijken / corrigeren de informatie uit de UZI credentials niet (human readable namen), dit moet bij de bron opgelost worden
> *Bij gebrek aan een generieke lokalisatiedienst wordt er een workaround toegepast waarbij handmatig wordt vastgelegd waar de patient in zorg is en waar dus informatie opgehaald kan worden

Reviewer: ```suggestion
* We sluiten niet aan op Mitz (maar gaan consent lokaal oplossen)
* Er is geen goed authenticatiemiddel voor de gebruiker beschikbaar (DEZI) dus werken we met een VC waarbij de organisatie obv een UZI certificaat wordt geauthenticeerd
* We implementeren niet alle mogelijke ZIB’s want die zijn niet beschikbaar in bronsystemen, maar hanteren een subset
* We passen geen DPoP security toe (RFC9449)  want dit is aanvullende beveiliging die ook in andere usecase niet ingevuld wordt
* We verrijken / corrigeren de informatie uit de UZI credentials niet (human readable namen), dit moet bij de bron opgelost worden
* Bij gebrek aan een generieke lokalisatiedienst wordt er een workaround toegepast waarbij handmatig wordt vastgelegd waar de patient in zorg is en waar dus informatie opgehaald kan worden
```
